src/Razor/test/testapps/Directory.Build.props

```diff
@@ -30,4 +30,12 @@
 
   <Import Project="After.Directory.Build.props" Condition="Exists('After.Directory.Build.props')" />
 
+  <PropertyGroup>
+    <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
+      $(RestoreSources);
+      https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
```

Comment:
It turns out that using the latest runtimes require "downloadDependencies". For example, if you look at the project.assets.json you see
```json
"frameworks": {
      "netcoreapp3.0": {
        "dependencies": {
          "FSharp.Core": {
            "target": "Package",
            "version": "[4.6.2, )"
          },
          "Microsoft.NETCore.App": {
            "include": "None",
            "suppressParent": "All",
            "target": "Package",
            "version": "[3.0.0-preview6-27804-01, )",
            "autoReferenced": true
          }
        },
        "imports": [
          "net461",
          "net462",
          "net47",
          "net471",
          "net472",
          "net48"
        ],
        "assetTargetFallback": true,
        "warn": true,
        "downloadDependencies": [
          {
            "name": "Microsoft.NETCore.App.Host.win-x64",
            "version": "[3.0.0-preview7-27819-07, 3.0.0-preview7-27819-07]"
          }
        ],
        "frameworkReferences": {
          "Microsoft.NETCore.App": {
            "privateAssets": "all"
          }
        }
      }
    }
  }
}
```

For now I'm working around the issue by adding the dotnet feed which contains non-released versions of this package. For more details and whether this is expected, see https://github.com/dotnet/core-sdk/issues/1958.